### DISCUSSION TITLE
ACE - explosif

### DISCUSSION THREAD

**Discussion by Fulgurofist** [acemod/ACE3#9207] [Q&A]
hello, I'm French and sorry for my English in advance...I have no problems with the "explosives" part of ACE mode when I install ACE on a server. explosives don't work. as soon as I want to fire it, whether it's through the timer, a handle-style trigger mechanism or a pressure release, nothing happens... whereas if I'm playing in LAN, hosting a game, everything works fine...
Thank you in advance for your help...

**Comment by rautamiekka** [comment]
How do each of the trigger options manifest as not working ?

**Reply by Fulgurofist** [reply]
Thank you for your answer. But sorry, I don't understand what you're asking me, sorry... I'm French and totally noob when it comes to programming, scripting, etc... Searching the forum I see that sometimes some ask to see the "newspaper", but I have no idea how it's done 😔
What I can tell you to try to be as clear as possible is that my server is a Nitrado. That I put the ACE mod, + the "bikey" file in the server's key folder.
But nothing to do, it only works when I play a solo or LAN game.
Maybe it comes from the game scenario itself, the "kpliberation"

**Reply by rautamiekka** [reply]
I mean:
- describe in detail what you do, and what you see, when you activate the timer.
- describe what you see and don't see when the timer runs out.
- describe in detail what you do to connect a remote activator, and what you see when you use the remote activator to explode the connected explosive.

If you still don't understand, I can't help. Someone who speaks French will have to help.

**Comment by Fulgurofist** [comment]
xplosives in the corresponding part, I specify that it does the same thing for each type of explosive that I choose.Then I take all the triggers possible by the ACE to see if it works with a...I press "win+ctrl" and manage to select the explosive, I put it on the ground, then I do "win" while looking at the explosive, then I select the means of firing. This tells me, for example, "code1" for the chosen explosive.Then I do "win+ctrl" to select the chosen means of firing, it offers me to explode everything, or to just choose code 1, but that I do one or the other, once done, nothing at all happens. And it's the same when I want to do by timer, after the specified time, nothing happens.I specify that it does that on any mission, even basic games, which are on the server.If I play in LAN, or solo, it works perfectly.Hoping to have been precise enough... And again thank you for your helpEnvoyé depuis mon téléphone Orange-------- Message original --------De : Jouni Järvinen ***@***.***>Date : mar. 6 juin 2023 à 00:39À : acemod/ACE3 ***@***.***>Cc : Fulgurofist ***@***.***>, Author ***@***.***>Objet : Re: [acemod/ACE3] ACE - explosif (Discussion #9207)
I mean:
describe in detail what you do, and what you see, when you activate the timer.describe what you see and don't see when the timer runs out.describe in detail what you do to connect a remote activator, and what you see when you use the remote activator to explode the connected explosive.
If you still don't understand, I can't help. Someone who speaks French will have to help.
